variables.tf

```diff
@@ -43,3 +43,9 @@ variable "github_actions_secret_ecr_secret_key" {
   default     = "ECR_AWS_SECRET_ACCESS_KEY"
   type        = string
 }
+
+variable "enable_ecr_env_secret" {
+  description = "Enable environment secret"
```

Comment:
Can it be "Enable github environment secret"?

Reply:
variable `enable_ecr_env_secret` is not used. Can you please remove if not used?